### ISSUE TITLE
Got an error within kafka-python while waiting for messages

### ISSUE THREAD

**Issue by kierkegaard13** [dpkp/kafka-python#840]
I got the error below while waiting for new messages. I'm not sure what's causing this and was wondering if this is some bug with kafka-python.

```
2016-10-04 15:11:02,416 mindfabric.services.message_handlers.osi_tag_handshake_message_handler INFO     Published ack to topic: sp_windows_service_pulljob_producer_invenergy
Traceback (most recent call last):
  File "/usr/local/bin/kafka-osi", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/site-packages/mindfabric/services/kafka_handshake_listener.py", line 92, in main
    listener.get_messages()
  File "/usr/local/lib/python3.5/site-packages/mindfabric/services/kafka_handshake_listener.py", line 60, in get_messages
    for message in self._consumer:
  File "/usr/local/lib/python3.5/site-packages/kafka/consumer/group.py", line 842, in __next__
    ret = self.poll(timeout_ms=self.config['consumer_timeout_ms'], max_records=1)
  File "/usr/local/lib/python3.5/site-packages/kafka/consumer/group.py", line 516, in poll
    records = self._poll_once(remaining, max_records)
  File "/usr/local/lib/python3.5/site-packages/kafka/consumer/group.py", line 552, in _poll_once
    records, partial = self._fetcher.fetched_records(max_records)
  File "/usr/local/lib/python3.5/site-packages/kafka/consumer/fetcher.py", line 323, in fetched_records
    if part.has_more():
  File "/usr/local/lib/python3.5/site-packages/kafka/consumer/fetcher.py", line 701, in has_more
    return self.message_idx < len(self.messages)
TypeError: object of type 'NoneType' has no len()
```

**Comment by dpkp** [comment]
Yes, it's a bug in master. Try the latest release, 1.3.1

**Comment by dpkp** [comment]
pushed fix in `9450a6b`

**Comment by kierkegaard13** [comment]
thanks for the help
